lib/msal-common/src/authority/Authority.ts

```diff
@@ -55,6 +58,8 @@ export class Authority {
     protected performanceClient: IPerformanceClient | undefined;
     // Correlation Id
     protected correlationId: string | undefined;
+    // Reserved tenant domain names that will not be replaced with tenant id
+    private static reservedTenantDomains: Set<string> = new Set([Constants.DEFAULT_COMMON_TENANT, "{tenant}", "{tenantid}"]);
```

Comment:
Super nitpicky - does this have to be a Set or can it be a basic array? Also, let's include the `consumers` and `organizations` reserved words here as well